qutebrowser/browser/webengine/certificateerror.py

```diff
@@ -0,0 +1,43 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
Needing a separate set of files for this is kind of unfortunate, but I didn't find a better location where it's isolated (webkit vs. webengine) and doesn't have circular import issues.